### PR TITLE
README Cleanup

### PR DESCRIPTION
**Pull request by arjansingh**
Still chugging along with this, but I figure early feedback is better than late feedback.

I have gone over every section up to "Disabling incompatible dependencies".

I still need to add FastBoot Request documentation also. Feedback is welcome. I will pick up again tomorrow.

Edit:

Alright I think this guy is ready for a full review. Have at it!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
-Currently, the set of Ember applications supported is extremely limited.
-As we fix more issues, we expect that set to grow rapidly. See [Known
-Limitations](#known-limitations) below for a full-list.
+## How It Works
 
-The bottom line is that you should not (yet) expect to install this add-on in
-your production app and have FastBoot work.
+FastBoot works by creating a Node virtual [sandbox](https://github.com/ember-fastboot/fastboot/blob/master/lib/ember-app.js#L50-L75) and [executing](https://github.com/ember-fastboot/fastboot/blob/master/lib/ember-app.js#L165-L232) your Ember Application within it.
+
+Most Ember applications should work out of the box. However, there are some idiomatic patterns of design that you have to be sure to follow. See [Known Limitations](#known-limitations) below for a full-list.
```
Comment (rtablada) [inline README.md]: I think maybe changing this to:

> Most Ember applications should work out of the box. However, there are some patterns that you will make sure your app runs with FastBoot. See [Known Limitations](#known-limitations) below for a full-list.

I would stay clear of `idiomatic` to make things less intimidating.
Especially in the docs/readme.

Comment (arjansingh) [inline README.md]: Good suggestions. Will it in the next pull.

Comment (arjansingh) [inline README.md]: My suggested text:

> Most Ember applications should work out of the box. However, there are some patterns that you be sure should follow to to guarantee that your application is fully FastBoot compatible. See [Known Limitations](#known-limitations) below for a full-list.

File: README.md [hunk 2]
```diff
@@ -1,15 +1,12 @@
...
+An Ember CLI Addon that allows you to render and serve Ember.js apps on the server. Using FastBoot, you can serve rendered HTML to browsers and other clients without requiring them to download JavaScript assets.
 
-Currently, the set of Ember applications supported is extremely limited.
-As we fix more issues, we expect that set to grow rapidly. See [Known
-Limitations](#known-limitations) below for a full-list.
+## How It Works
 
-The bottom line is that you should not (yet) expect to install this add-on in
-your production app and have FastBoot work.
+FastBoot works by creating a Node virtual [sandbox](https://github.com/ember-fastboot/fastboot/blob/master/lib/ember-app.js#L50-L75) and [executing](https://github.com/ember-fastboot/fastboot/blob/master/lib/ember-app.js#L165-L232) your Ember Application within it.
```
Comment (pwfisher) [inline README.md]: You're referencing specific lines but using a branch URL and not a specific SHA. This will break.

File: README.md [hunk 3]
```diff
@@ -1,15 +1,12 @@
...
+
+Most Ember applications should work out of the box. However, there are some patterns that you be sure should follow to to guarantee that your application is fully FastBoot compatible. See [Tips and Tricks](#tips-and-tricks) below for a full-list.
```
Comment (pwfisher) [inline README.md]: "you be sure should follow" is garbled.

File: README.md (hunk 2 above)
Comment (tomdale) [inline README.md]: What's the difference between a virtual sandbox and a regular sandbox? 😉 

Also, seems strange to link deep into the code from the readme like this? I haven't seen that before. It shouldn't be important for people to understand the internals to use it, but this might send that message.

Comment (arjansingh) [inline README.md]: Can remove deep link and change it just be sandbox

**Comment by arjansingh** [comment]
Closed in favor of https://github.com/ember-fastboot/fastboot-website/pull/41
